x-pack/plugins/uptime/server/legacy_uptime/lib/lib.ts

```diff
@@ -225,17 +227,25 @@ export class UptimeEsClient {
       indices = settings?.heartbeatIndices || '';
       syntheticsIndexRemoved = settings.syntheticsIndexRemoved ?? false;
     }
-    if (
-      this.isLegacyAlert &&
-      !indices.includes('synthetics-') &&
-      (syntheticsIndexRemoved || !settingsChangedByUser)
-    ) {
+    if (indices.includes('synthetics-')) {
+      return indices;
+    }
+    const appendSyntheticsIndex = shouldAppendSyntheticsIndex(this.stackVersion);
+
+    if (appendSyntheticsIndex && (syntheticsIndexRemoved || !settingsChangedByUser)) {
       indices = indices + ',synthetics-*';
     }
     return indices;
   }
 }
 
+export const shouldAppendSyntheticsIndex = (stackVersion?: string) => {
+  if (!stackVersion) {
+    return false;
+  }
+  return semver.lte(stackVersion, '8.10.0');
```

Comment:
Should it be `semver.lt(stackVersion, '8.10.0')`? 

Referring the AC:
> In 8.10 we need to remove synthetics-* for any new alerts that may be created in Uptime

Reply:
yes, you are right